react/StoreWrapper.js

```diff
@@ -152,7 +152,7 @@ const StoreWrapper = ({ children, CustomContext }) => {
         script={[
           supportsServiceWorker && {
             type: 'text/javascript',
-            src: `${rootPath}/pwa/workers/register.js${queryMatch}${
+            src: `${rootPath}/serviceWorkerRegister.js${queryMatch}${
```

Comment:
We should probably change this route, so that it's not in the website's root.
Something like `/service-worker/register.js`